artcommon/artcommonlib/bigquery.py

```diff
@@ -74,15 +78,11 @@ def insert(self, names: typing.List[str], values: list) -> None:
         https://github.com/googleapis/python-bigquery-storage/blob/main/samples/snippets/append_rows_proto2.py
         """
 
-        assert isinstance(names, list)
-        assert isinstance(values, list)
-        assert names and values, 'Names and values cannot be empty lists'
```

Comment:
an empty check on items would be nice here

Reply:
These are gone, we now feed in a dict. If empty, the resulting query will fail throwing an exception. I think this is enough for us to spot errors